x/did/types/genesis.pb.go

```diff
@@ -83,30 +135,79 @@ func (m *GenesisState) GetFeeParams() *FeeParams {
-	0x4a, 0x0f, 0x40, 0xc1, 0xac, 0x16, 0x15, 0x9a, 0x46, 0x48, 0x0c, 0x49, 0x4c, 0x92, 0x71, 0xee,
-	0x83, 0x82, 0x87, 0x4d, 0x16, 0x5c, 0xd1, 0xfd, 0x7f, 0x08, 0xb4, 0x34, 0xe1, 0x4e, 0x3c, 0x4a,
-	0x26, 0x59, 0xcc, 0xb7, 0x2f, 0xe1, 0x53, 0x05, 0x53, 0x2d, 0x9f, 0x95, 0x2d, 0xef, 0xd1, 0x0a,
-	0x10, 0x56, 0xe4, 0x7b, 0xe0, 0x32, 0x13, 0x5c, 0x52, 0x5a, 0x20, 0xce, 0x1a, 0xb1, 0x14, 0x95,
-	0x09, 0x47, 0x31, 0x49, 0x26, 0xd9, 0xd1, 0x70, 0xfc, 0x16, 0xf1, 0xd1, 0xd5, 0xf9, 0xb8, 0xd8,
-	0x7c, 0x6f, 0xae, 0xbf, 0x5b, 0x46, 0xd6, 0x2d, 0x23, 0xbf, 0x2d, 0x23, 0x9f, 0x1d, 0xf3, 0xd6,
-	0x1d, 0xf3, 0x7e, 0x3a, 0xe6, 0xbd, 0x9c, 0x2d, 0x94, 0x2d, 0xdf, 0xe6, 0x5c, 0xea, 0x2a, 0xed,
-	0x3d, 0xdd, 0x7b, 0x5e, 0x6b, 0xc0, 0xf4, 0xdd, 0x49, 0xdb, 0x8f, 0x06, 0xcd, 0x7c, 0xd7, 0x49,
-	0x5f, 0xfc, 0x05, 0x00, 0x00, 0xff, 0xff, 0xe3, 0x5d, 0xa2, 0xe1, 0x53, 0x01, 0x00, 0x00,
+	// 325 bytes of a gzipped FileDescriptorProto
+	0x1f, 0x8b, 0x08, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0xff, 0x6c, 0x50, 0x41, 0x4b, 0xf3, 0x40,
+	0x10, 0xed, 0x7e, 0x85, 0x4f, 0xbb, 0x4d, 0x45, 0x72, 0xd0, 0xda, 0xc3, 0x12, 0x2a, 0x62, 0x2f,
+	0x26, 0x10, 0xc1, 0x8b, 0xa7, 0x4a, 0xd1, 0x93, 0x22, 0x29, 0x28, 0x78, 0x09, 0xdb, 0xcc, 0xb4,
+	0x5d, 0xb0, 0xd9, 0xd8, 0x1d, 0x83, 0xfe, 0x0b, 0x7f, 0x89, 0xbf, 0xc3, 0x63, 0x8f, 0x1e, 0xa5,
+	0xfd, 0x23, 0xe2, 0x6e, 0xab, 0x16, 0xbc, 0x0c, 0xc3, 0x7b, 0x8f, 0xf7, 0x66, 0x1e, 0x6f, 0x65,
+	0x63, 0x7c, 0x80, 0x08, 0x14, 0x44, 0x65, 0x1c, 0x8d, 0x30, 0x47, 0xa3, 0x4c, 0x58, 0x4c, 0x35,
+	0x69, 0xdf, 0xb3, 0x5c, 0x08, 0x0a, 0xc2, 0x32, 0x6e, 0xed, 0xad, 0x29, 0x41, 0x01, 0xe8, 0xcc,
+	0x09, 0x5b, 0x3b, 0x6b, 0xd4, 0x10, 0xd1, 0xe1, 0xed, 0x92, 0x6f, 0xf7, 0x14, 0xf4, 0x74, 0x76,
+	0x83, 0x53, 0xa3, 0x74, 0xde, 0x47, 0xf2, 0x0f, 0xf8, 0xd6, 0xbd, 0x24, 0x34, 0x94, 0x96, 0x0e,
+	0x6c, 0xb2, 0x80, 0x75, 0x6a, 0x49, 0xc3, 0xa1, 0x4b, 0xa5, 0x7f, 0xca, 0x37, 0x41, 0x41, 0x0a,
+	0x3a, 0x33, 0xcd, 0x7f, 0x41, 0xb5, 0x53, 0x8f, 0x83, 0xf0, 0xf7, 0x39, 0xa1, 0x33, 0xbe, 0x55,
+	0x34, 0xbe, 0x44, 0x92, 0x20, 0x49, 0x26, 0x1b, 0x60, 0x31, 0xd3, 0x7e, 0x65, 0xdc, 0xbb, 0x70,
+	0xaf, 0xf4, 0x49, 0x12, 0xfa, 0xfb, 0xbc, 0xf1, 0xe5, 0x96, 0xcb, 0x09, 0x9a, 0x42, 0x66, 0xb8,
+	0xcc, 0xf4, 0x40, 0xc1, 0xd5, 0x0a, 0xf3, 0xbb, 0xdc, 0x5b, 0x9e, 0x94, 0x1a, 0xa4, 0x55, 0xac,
+	0xf8, 0x2b, 0xf6, 0xe7, 0x9f, 0xa4, 0x5e, 0x7e, 0xef, 0xc6, 0x3f, 0xe1, 0x7c, 0x88, 0x98, 0x16,
+	0x72, 0x2a, 0x27, 0xa6, 0x59, 0x0d, 0x58, 0xa7, 0x1e, 0xef, 0xae, 0x1b, 0x9c, 0x23, 0x5e, 0x5b,
+	0x3a, 0xa9, 0x0d, 0x57, 0xeb, 0x59, 0xf7, 0x6d, 0x2e, 0xd8, 0x6c, 0x2e, 0xd8, 0xc7, 0x5c, 0xb0,
+	0x97, 0x85, 0xa8, 0xcc, 0x16, 0xa2, 0xf2, 0xbe, 0x10, 0x95, 0xbb, 0xc3, 0x91, 0xa2, 0xf1, 0xe3,
+	0x20, 0xcc, 0xf4, 0x24, 0x72, 0x2d, 0xdb, 0x79, 0x94, 0x6b, 0xc0, 0xe8, 0xc9, 0x56, 0x4e, 0xcf,
+	0x05, 0x9a, 0xc1, 0x7f, 0x5b, 0xf9, 0xf1, 0x67, 0x00, 0x00, 0x00, 0xff, 0xff, 0xed, 0xeb, 0xc7,
+	0x91, 0xd1, 0x01, 0x00, 0x00,
+}
+
```

Comment:
What is this and why does it have such a weird name?

Reply:
This is internals of generated proto. Looks like it keeps bytes of the original `.proto` file. Hard to say why they do it.

Reply:
It's a way to pass `.proto` field definitions typed to descriptors, as bytes and parse them later, during the decoding process regardless of language.